sbt-app/src/sbt-test/tests/setup-cleanup/test

```diff
@@ -39,7 +39,7 @@ $ absent tested
 $ absent cleanup
 
 # without Setup configured, the setup file won't exist and the test will fail
--> testQuick
+-> testFull
```

Comment:
Same here - this changes the test type. Looking at the rest of the file it seems like it's fine, the whole thing looks like it's doing full tests with this change. But wanted to make sure this was your intent.